wcfsetup/install/files/lib/system/tagging/TagEngine.class.php

```diff
@@ -111,10 +113,12 @@ public function deleteObjectTags($objectType, $objectID, $languageID = null)
     {
         $objectTypeID = $this->getObjectTypeID($objectType);
 
-        $sql = "DELETE FROM wcf" . WCF_N . "_tag_to_object
-                WHERE       objectTypeID = ?
-                        AND objectID = ?
-                        " . ($languageID !== null ? "AND languageID = ?" : "");
+        $sql = "DELETE FROM wcf" . WCF_N . "_tag_to_object tag_to_object
```

Comment:
Will have to recheck this query. I believe I forgot to adjust the code after testing in MySQL CLI.